recipes/python/config/access_hosts_api_usecases.py

```diff
@@ -0,0 +1,165 @@
+import sys
+import login.login_api as login_api
+import config.access_hosts_api as access_hosts_api
+
+nbmaster = ""
+username = ""
+password = ""
+domainName = ""
+domainType = ""
+hostName = ""
+
+EXCLUDE_CONFIG_NAME = "Exclude"
+
+def print_disclaimer():
+    print("\n-------------------------------------------------------------------------------------------------")
+    print("--                          This script requires Python3.5 or higher.                          --")
+    print("--    The system where this script is run should have Python 3.5 or higher version installed.  --")
+    print("-------------------------------------------------------------------------------------------------")
+    print("The script requires 'requests' library to make the API calls.")
```

Comment:
It's pointless to mention this at run time. If you have the module, then the command will run and you don't need to be reminded that you need a module you already have. If you don't have the module, then the script won't even run, so this line of code wouldn't run anyway.